src/sass/partials/global/_mixins.scss

```diff
@@ -128,7 +138,7 @@
 }
 
 @mixin radio {
-  display: none;
+  transform: translateX(.5em);
```

Comment:
This doesn't entirely hide the radio buttons. I tried with `translateX(.25em) translateY(-.15em)` and that put the radio element entirely behind the custom styling. With `.5em` it peeks out the bottom right, which is very prominent on iOS

**.5em**
<img width="384" alt="screen shot 2016-09-01 at 1 48 08 pm" src="https://cloud.githubusercontent.com/assets/377188/18178081/e99425aa-704a-11e6-9dae-51c092646b57.png">
<img width="285" alt="screen shot 2016-09-01 at 1 48 03 pm" src="https://cloud.githubusercontent.com/assets/377188/18178082/e99ed2ac-704a-11e6-96fc-7456da0e6505.png">

**.25em .15em**
<img width="328" alt="screen shot 2016-09-01 at 1 48 31 pm" src="https://cloud.githubusercontent.com/assets/377188/18178101/f97774d6-704a-11e6-975d-ff23ef2e3037.png">
<img width="357" alt="screen shot 2016-09-01 at 1 48 15 pm" src="https://cloud.githubusercontent.com/assets/377188/18178102/f97b5bdc-704a-11e6-809e-75002bbd370a.png">